src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -262,15 +262,9 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
         >
           <div>
             <Nav bsClass="nav nav-tabs nav-tabs-pf">
-              <NavItem eventKey="info">
-                <div>Overview</div>
-              </NavItem>
-              <NavItem eventKey="traffic">
-                <div>Traffic</div>
-              </NavItem>
-              <NavItem eventKey="metrics">
-                <div>Inbound Metrics</div>
-              </NavItem>
+              <NavItem eventKey="info">Overview</NavItem>
+              <NavItem eventKey="traffic">Traffic</NavItem>
+              <NavItem eventKey="metrics">Inbound Metrics</NavItem>
```

Comment:
in another file in this PR Inbound metrics key is in_metrics and here it is just metrics. isn't it a bit confusing/not consistent?

Reply:
Totally agree I am think about move all of this in a component to reuse it and avoid this, but we should do it in another PR

Reply:
no objections from my side